Failed to fetch remote resources from mwdb web

**Environment information**
* MWDB version : 2.8.2
* Installation method:
  - [ ] mwdb.cert.pl service
  - [x] From PyPi (`pip install mwdb-core`)
  - [ ] From docker-compose
  - [ ] Other (please explain)
* Plugins installed: None 

**Behaviour the bug (what happened?)**

I added a remote instance to try the remote instance feature. I followed the documentation and added the cert.pl mwdb instance with my own API key (I have access). I can switch to the remote instance, the dashboard view is accurate (same list of files as in mwdb.cert.pl). But, when I open a file/object, an error is raised: "Unknown remote instance name ('null')". Maybe the remote_name is getting lost somewhere between the web request and the API request?

```
[ERROR] Thread-15 - service.error_router:71 - 404 Not Found: Unknown remote instance name ('null')
[INFO] Thread-15 - app.log_request:146 - request - path:/api/remote/null/api/file/<redacted> - arguments:{} - method:GET - status:404 - response_time:0:00:00.011607 - response_size:53
```

The Configs tab has the same issue.

**Expected behaviour**

The file summary should appear, and I should be able to Pull or Download the information.

**Reproduction Steps**

mwdb.ini:
 
```
[mwdb]

[...]
remotes = mwdbcertpl

[...]

[remote:mwdbcertpl]
url = https://mwdb.cert.pl
api_key=<redacted>
```

Reload mwdb-core, switch to remote view mwdbcertpl, click on the first file.

**Screenshots**

![image](https://user-images.githubusercontent.com/15208659/212397353-8c5ce11c-7a07-41da-935c-6e631bee8feb.png)

**Additional context**

I tried to poke around and found that remote_name argument of RemoteApiResource.dorequest(...) is null.

--- a/mwdb/web/src/commons/remotes/index.js
+++ b/mwdb/web/src/commons/remotes/index.js
@@ -1,4 +1,4 @@
-import { useLocation, matchPath, useMatch } from "react-router-dom";
+import { useLocation, matchPath } from "react-router-dom";
 
 export function useRemote() {
     // Returns current remote name or undefined if current view is local

--- a/mwdb/web/src/commons/remotes/index.js
+++ b/mwdb/web/src/commons/remotes/index.js
@@ -1,9 +1,13 @@
-import { useMatch } from "react-router-dom";
+import { useLocation, matchPath, useMatch } from "react-router-dom";
 
 export function useRemote() {
     // Returns current remote name or undefined if current view is local
-    const match = useMatch("/remote/:remote");
-    return match && match.params.remote;
+    const location = useLocation();
+    const remotePath = matchPath(
+        { path: "/remote/:remote", end: false },
+        location.pathname
+    );
+    return remotePath && remotePath.params.remote;
 }
 
 export function useRemotePath() {

